linux/vhba_hot_add_remove/freebsd_create_disk_partition.yml

```diff
@@ -51,6 +51,22 @@
   ansible.builtin.set_fact:
     partition_device_path: "/dev/{{ partition_name }}"
 
+- name: "Get disk partition info"
```

Comment:
Please move `linux/vhba_hot_add_remove/freebsd_create_disk_partition.yml` to `linux/utils/freebsd_create_disk_partition.yml`, and update include_tasks paths affected.